packages/odyssey-react-mui/src/Checkbox.tsx

```diff
@@ -28,8 +28,29 @@ import {
   getControlState,
 } from "./inputUtils";
 import { FieldComponentProps } from "./FieldComponentProps";
+import { type FeatureTestSelector } from "./test-selectors";
 import { Typography } from "./Typography";
 
+export const CheckboxTestSelectors = {
+  // feature: {
```

Comment:
leaving as comment because we will eventually convert this as either a label or a different selector as discussed with @KevinGhadyani-Okta

Reply:
Can we commented this as:
```suggestion
  // label: ["hint"]
```
That'll make it easier in the future because the API won't match the feature API.